### DISCUSSION TITLE
Multiple document windows

### DISCUSSION THREAD

**Discussion by instancezero** [vkbo/novelWriter#1729] [Ideas]
Context: I'm in the process of preparing to migrate from Windows to Linux (likely Ubuntu) and am currently a Scrivener user. It looks like Scrivener under Wine is challenging at best so I'm looking for alternatives, and NovelWriter does almost everything. Great job.

I'm what's known as a "plotter". I tend to develop my stories from a broad outline, refining in many iterations to add detail until I'm at the point where I have a roadmap for the whole work. Then I use the roadmap to write each scene, in theory progressing from beginning to end. Except it's never that simple. There's a lot of back and forth, sometimes new things mess with the plot. Often I need to check to make sure what I'm writing doesn't conflict with the overall plan.

In Scrivener, I have two document windows open, the work in progress on the left; the plot outline on the right, and the outline is locked so that navigating the project tree only changes the left window. Is this anywhere near doable? The plot window doesn't need to be editable (although that can be useful), I just want to be able to scroll through and reverence a fairly lengthy document without losing my edit window.

**Comment by instancezero** [comment]
I just discovered that "View Document" using the plot document looks like it will achieve what I need. This is great stuff!

**Reply by vkbo** [reply]
Yes, that is precisely what the view panel is for. You can also middle-click a document in the project tree to view it if your mouse has that option, or press Ctrl+R when the project tree has focus (pressing Ctrl+T will shift focus to the project tree if not).

I am a "plotter/outliner" too, so I designed novelWriter with that possibility in mind. As a next step, you can also outline all your core scenes in one or more documents, including chapters if yo wish, and then split them up into individual chapter and scene documents later on (Right-click -> Transform -> Split). I often outline my core structure in one document per act, before splitting.
